esrally/chart_generator.py

```diff
@@ -1195,21 +1330,40 @@ def load_track(cfg, name=None):
     return track.load_track(cfg)
 
 
-def generate_index_ops(chart_type, race_configs, environment):
+def generate_index_ops(chart_type, race_configs, environment, logger):
     idx_race_configs = list(filter(lambda c: "indexing" in c.charts, race_configs))
+    for race_conf in idx_race_configs:
+        logger.debug("Gen index visualization for race config with name:[{}] / label:[{}] / flavor: {} / license: {}".format(
+            race_conf.name, race_conf.label, race_conf.flavor, race_conf.lic))
+    charts = []
+
     if idx_race_configs:
-        title = format_title(environment, idx_race_configs[0].track, "indexing-throughput")
-        return [chart_type.index(environment, idx_race_configs, title)]
-    else:
-        return []
+        title = format_title(chart_type, environment, race_configs[0].track, "indexing-throughput", flavor=race_configs[0].flavor)
+        charts = [chart_type.index(environment, idx_race_configs, title)]
+    return charts
 
 
-def filter_string(environment, race_config):
+def filter_string_time_series(environment, race_config):
```

Comment:
This should now be a method on the respective classes if we need to do type tests.